docs/mkdocs.yaml

```diff
@@ -0,0 +1,90 @@
+site_name: niri
+docs_dir: wiki
+site_url: https://chinatsu.github.io/niri/ # TODO before merging: https://yalter.github.io/niri
+repo_url: https://github.com/chinatsu/niri # TODO before merging: https://github.com/YaLTeR/niri
+edit_uri: edit/main/docs/wiki/
+use_directory_urls: false
+theme:
+  name: material
+  features:
+    - navigation.instant
+    - search.suggest
+    - content.code.copy
+    - content.action.edit
+  palette:
+    scheme: slate
+    primary: black
+markdown_extensions:
+  - github-callouts
+  - pymdownx.highlight:
+      anchor_linenums: true
+      line_spans: __span
+      pygments_lang_class: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences
+  - admonition
+  - pymdownx.details
+  - pymdownx.keys
+  - attr_list
+plugins:
+  - search
+hooks:
+  - hooks/shortcodes.py
+  - hooks/remove-must-fail.py
+extra_css:
+  - _assets/stylesheets/niri.css
+strict: true
+validation:
+  nav:
+    omitted_files: warn
+    not_found: warn
+    absolute_links: relative_to_docs
+  links:
+    not_found: warn
+    anchors: warn
+    absolute_links: relative_to_docs
+    unrecognized_links: warn
+not_in_nav: |
+  _Sidebar.md
+  Configuration:-Overview.md
+  index.md
+# ah, wouldn't it be nice if we could autogenerate this with wiki/_Sidebar.md
```

Comment:
Can't we?

Reply:
Should be a simple Python script. Don't need to parse full MD or anything, just autogenerate one from the other the simplest way possible

Reply:
probably, yeah. but for now i think i'd prefer to leave the navigation configured like this, even though it means we'd have to change it in two places if we do decide to move pages around. 

(ps: i'm leaving conversations answered with my opinion answer unresolved, so if you feel my answer's sufficient feel free to hit that resolve button!)